bigtop-packages/src/charm/zookeeper/layer-zookeeper/actions/restart

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+sys.path.append('lib')  # Add our helpers to our path.
+
+from charmhelpers.core import hookenv
+from charms.layer.zookeeper import Zookeeper
+
+LOG = hookenv.log
+
+LOG("starting restart handler.")
+
+def main():
+    LOG("starting restart handler main.")
+    hookenv.status_set('waiting', 'Restarting ...')
+    zookeeper = Zookeeper()
+    zookeeper.install()
```

Comment:
This should be predicated on the `zookeeper.started` state and should `action_fail` if it's not set.